mobile/examples/super_resolution/ios/ORTSuperResolution/ORTSuperResolution.xcodeproj/project.pbxproj

```diff
@@ -7,12 +7,13 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		510A53D6296DE390000DB268 /* pt_super_resolution_with_pre_post_processing_opset16.onnx in Resources */ = {isa = PBXBuildFile; fileRef = 510A53D5296DE390000DB268 /* pt_super_resolution_with_pre_post_processing_opset16.onnx */; };
```

Comment:
Should this file be under source control?

Reply:
this is the older opset16 version model, do we still also want to keep that model in the project?

also fyi .pbxproj file updates are auto-generated after updating the ios app.

Reply:
I'd prefer we only have the opset 18 model given that will have significantly better output.

Reply:
> also fyi .pbxproj file updates are auto-generated after updating the ios app.

That's why I wondered the pbxporj file should be under version control? If it gets re-generated. Not a biggie